sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -255,9 +255,9 @@ The following group algorithms support `ballot_group`, `fixed_size_group`,
 
 A ballot-group is a non-contiguous subset of a group, representing a collection
 of all work-items in the group that share the same value of some predicate.
-Ballot-groups are always created in a range of two: the first ballot-group
-contains all work-items where the predicate is true, and the second
-ballot-group contains all work-items where the predicate is false.
+Ballot-groups are always created in a range of two: the one ballot-group
```

Comment:
I would also remove the "the", though I think both are correct.
```suggestion
Ballot-groups are always created in a range of two: one ballot-group
```

Reply:
I agree that "the" should be removed here.